config/application.rb

```diff
@@ -61,6 +61,10 @@ class Application < Rails::Application
       user_name: ENV.fetch("SMTP_USERNAME", nil),
       password: ENV.fetch("SMTP_PASSWORD", nil)
     }.compact
+
+    config.upstream_repo = ENV.fetch("UPSTREAM_REPO", "https://github.com/manyfold3d/manyfold")
+    config.app_version = ENV.fetch("APP_VERSION", "unknown")
```

Comment:
We should do the split here to remove the docker prefix that gets added on.

```suggestion
    config.app_version = ENV.fetch("APP_VERSION", "unknown").split(":")[-1]
```